src/include/rbd/librbd.h

```diff
@@ -144,6 +144,8 @@ enum {
   RBD_IMAGE_OPTION_JOURNAL_ORDER = 5,
   RBD_IMAGE_OPTION_JOURNAL_SPLAY_WIDTH = 6,
   RBD_IMAGE_OPTION_JOURNAL_POOL = 7,
+  RBD_IMAGE_OPTION_FEATURES_SET = 8,
```

Comment:
I am not sure I understand the purpose of these new options.  If ```RBD_IMAGE_OPTION_FEATURES``` is set, you should use them -- otherwise use the defaults.